tests/auto/integration/tst_integration.cpp

```
@@ -47,6 +47,14 @@ static QString findGpg() {
   return QStandardPaths::findExecutable("gpg");
 }
 
+static QString findGpgconf() {
```

Clang (14.0.6)

[error] use a trailing return type for this function

(modernize-use-trailing-return-type,-warnings-as-errors)